_replication/scheduling/advanced-cron-scheduling.md

```diff
@@ -0,0 +1,334 @@
+            {% endfor %}
+    {% endfor %}
+</table>
+
+
+---
+
+## Example replication schedules {#examples}
+
+{% capture resource-callout %}
+**Want to test an expression?** Use [{{ cron.resource-names.crontab }}]({{ cron.resource-urls.crontab }}){:target="new"} to test expressions and see the schedule that will be created.
+{% endcapture %}
+
+{% include tip.html content=resource-callout %}
+
+Based on scheduling feedback we've collected, we've put together a list of some of the most commonly requested replication schedules. The cron expressions in the table below can be copied and pasted directly into an integration's settings page in Stitch.
+
+**Note**: When creating your own cron expressions, keep in mind that there are some [limitations](#limitations).
+
+<table class="attribute-list">
+    <tr>
+        <td>
+            <strong>
+                Expression
+            </strong>
+        </td>
+        <td>
+            <strong>
+                Translation
+            </strong>
+        </td>
+    </tr>
+    {% for example in cron.examples %}
+        <tr>
+            <td>
+                {{ example.expression | markdownify }}
+            </td>
+            <td>
+                {{ example.translation }}
+            </td>
+        </tr>
+    {% endfor %}
+</table>
+
+---
+
+## Limitations of cron scheduling {#limitations}
+
+While using cron expressions will give you the most control over your integrations' replication schedules, there are some limitations to keep in mind:
+
+1. **Advanced scheduling isn't available for all integrations**. Some database integrations don't currently support advanced scheduling. We are working on converting these integrations into Singer-powered taps, at which point advanced scheduling will be available.
+
+2. **Days of the week in specific months can't be specified** due to the implementation of cron Stitch uses. POSIX (the name of the cron library) [has a unique method for specifying days](#specifying-days).
+
+   For example: The expression `* * * 1 1` translates to _"every day in the first month (January) and the first day of every week (Monday)"_, **not** _"every Monday in January"_.
+
+3. **Advanced Scheduling can only be used to whitelist extraction start times**. This means that a job could start during a whitelisted time period but continue running beyond that window, depending on the duration of the extraction.
+
+   For example: An integration has a schedule that tells it to run every 20 minutes between the hours of noon and 2:00PM, starting at noon.
+
+   On average, extractions for this integration take between 2-5 minutes. However, the extraction that starts at 1:40PM takes longer than average, causing the job to continue running even after the 2:00PM mark:
+
+   ![Grid showing a timeline of extraction starts during whitelisted extraction hours]({{ site.baseurl }}/images/replication/cron-extraction-whitelist-example.png)
+
+---
+
+## Troubleshooting {#troubleshooting-cron-errors}
```

Comment:
Much of this will change once the new cron library is put in place.

Reply:
Yep, I saw you mentioned that in Slack - do you have a lead on what those errors will say yet?

Reply:
I don't have much specific information but the error generated _should_ allow us to say something like `5 fields are required (you specified '4')` or `6)` and `The hour field must be an integer between 0 and 23 (you specified '24')` or `'24,1-10')` or `'Yah/tva,.pn#')`.

Reply:
I'll say this too, we should have the ability to generate whatever kind of error messages you think would be best, and you/lauren/mark/harrison are probably much better at phrasing error messages. So I'd make these look however you want them to look and whatever engineer works on it will make it happen. The error messages before were more because we were hoping someone else would do the error detection. Now we've been forced to do all of it so we can make it be whatever we want.